test/integration.js_10.snap

```diff
@@ -255,13 +255,13 @@ exports[`c8 report generates report from existing temporary files 1`] = `
 ",--------------------------|---------|----------|---------|---------|--------------------------------
 File                      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s              
 --------------------------|---------|----------|---------|---------|--------------------------------
-All files                 |   75.57 |    60.53 |   66.67 |   75.57 |                                
+All files                 |   75.51 |    59.49 |   67.65 |   75.51 |                                
```

Comment:
Looks like some of these numbers changed as we use `test/*.js` as the glob. So I updated the snap and adjusted the threshold in a test to keep the spirit of the test correct.